velox/exec/Task.cpp

```diff
@@ -935,6 +936,32 @@ void Task::addHashJoinBridgesLocked(
   }
 }
 
+void Task::addCustomJoinBridgesLocked(
+    uint32_t splitGroupId,
+    const std::vector<std::shared_ptr<const core::PlanNode>>& planNodes) {
+  auto& translators = Operator::translators();
+  if (0 == translators.size()) {
```

Comment:
nit: `if (translators.empty())` might be easier to read

Reply:
move the logic to joinBridgeFromPlanNode method in Operator.